### PR TITLE
Day 5 in Go

### PR DESCRIPTION
**Pull request by tholok97**
Didn't update the README counter, as I realize it will be hard to keep it up to date with everyone incrementing it in their PRs.. Guess it's something we'll have to manually update every once in a while?

### CODE REVIEW

**Comment by Arxcis** [comment]
> Didn't update the README counter, as I realize it will be hard to keep it up to date with everyone incrementing it in their PRs.. Guess it's something we'll have to manually update every once in a while?

I will do it once per day. So you don't have to do it :+1:

File: days/day-05/solutions/tholok97.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	rowUpper := 127
+
+	for _, c := range boardingPass[:7] {
+		switch c {
+		case 'F':
+			rowUpper = rowLower + (rowUpper-rowLower)/2
+		case 'B':
+			rowLower = rowLower + (rowUpper-rowLower)/2 + 1
+		default:
+			panic("what??")
```
Comment (Arxcis) [inline days/day-05/solutions/tholok97.go:61]: Hihihi :+1: What!!! Approved!

**Review by Arxcis — APPROVED** [review]

**Comment by Arxcis** [comment]
> I will do it once per day. So you don't have to do it +1

I think we will have to automate it at one point. It takes a bit of time to update to update the numbers, and it is error prone.

**Comment by tholok97** [comment]
> > I will do it once per day. So you don't have to do it +1
> 
> I think we will have to automate it at one point. It takes a bit of time to update to update the numbers, and it is error prone.

Should be possible. `for each day, count number of each of '*.go', '*.js etc` :)

> We have the technology
> - Jonas, 2020

**Pull request merged**